imap/jmap_calendar.c

```diff
@@ -1201,6 +1201,7 @@ struct setcalendar_props {
     int isVisible;
     int isSubscribed;
     int transp;
+    bool ext_props; // request is setting externally-stored properties
```

Comment:
This could be more clearly named as a boolean, e.g. "sets_ext_props" or "has_ext_props"

Reply:
Done